src/main/java/org/team1540/robot2025/subsystems/Superstructure.java

```diff
@@ -0,0 +1,30 @@
+package org.team1540.robot2025.subsystems;
+
+import org.littletonrobotics.junction.AutoLogOutput;
+import org.team1540.robot2025.subsystems.arm.Arm;
+import org.team1540.robot2025.subsystems.elevator.Elevator;
+import org.team1540.robot2025.subsystems.grabber.Grabber;
+import org.team1540.robot2025.subsystems.intake.CoralIntake;
+
+public class Superstructure {
```

Comment:
Can we come up with a better name for this? Current name seems overly general and doesn't specify what it is. My understanding is that it is meant to coordinate the four non-dt subsystems but there would still be functions (deploy intake, zero elevator, etc) not handled by it.

Reply:
Deploy intake is handled by this, and elevator zeroing should be too, so the arm doesn't crash into things while zeroing occurs. Also, I just took the most commonly used name for such a class, and I think it makes enough sense.